ui/src/app/pages/login/account-creation/account-creation-survey.spec.tsx

```diff
@@ -49,6 +51,22 @@ it('should handle error when creating an account', async() => {
 
   const wrapper = mount(<AccountCreationSurvey {...props} />);
 
+  wrapper.find('[data-test-id="checkbox-race-PREFER_NO_ANSWER"]').at(1).simulate('change', {target: {checked: true}})
```

Comment:
I understand this test code is not ideal and can be refactored into better utils. I created ticket [RW-6725](https://precisionmedicineinitiative.atlassian.net/browse/RW-6275) to address this
I am optimizing for time to get this released and would prefer to address issues here in the follow-up.